src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -717,8 +728,20 @@ private void initializeInputPreloadExecutorService() {
     }
 
     private Void logMessageOnBatchCompletion(final BatchCompletionCallbackFunctionArgument arg) {
-        progressMeter.update(null);
         logger.info("Done importing batch " + arg.batchCount + "/" + arg.totalBatchCount);
+        logger.info("List of samples imported in batch " + arg.batchCount + ":");
```

Comment:
I'm not sure we want to output the list of every sample in the batch?  That seems very verbose.

Reply:
I actually requested this -- without a progress meter, knowing which samples you're on at least gives you some sense of where you are, and if something goes wrong you know which samples might be responsible.

Reply:
Maybe this should be debug output?

Reply:
Sure, I can make this debug if that's the consensus.

Reply:
Changed list of sample to debug